core/components/menu/menuitem.js

```diff
@@ -176,23 +171,7 @@ Blockly.MenuItem.prototype.setCheckable = function(checkable) {
  * @package
  */
 Blockly.MenuItem.prototype.setChecked = function(checked) {
```

Comment:
Why not do the same thing with ``setEnabled``? If I'm reading between the lines, the suggestions here is that you shouldn't be able to call setChecked on a menu item after it's been rendered (unless you call render) yourself.

I'm okay with that as we don't have a use for that in Blockly, but I don't think we have a use for that for setEnabled either.

Reply:
Oh, good catch.  Deleting all that junk in setEnabled.